devops/actions/run-tests/benchmark/action.yml

```diff
@@ -42,6 +42,12 @@ runs:
   # composite actions don't make use of 'name', so copy-paste names as a comment in the first line of each step
   using: "composite"
   steps:
+  - name: Install numactl
```

Comment:
perhaps install numactl in the step `- name: Install perf in version matching the host kernel`
... and probably rename to something more generic, like "install additional bench dependencies", plus make a separate group for this installation